process/transaction/shardProcess.go

```diff
@@ -800,12 +790,15 @@ func (txProc *txProcessor) executeFailedRelayedTransaction(
 		return err
 	}
 
+	//TODO: check if the totalFee can be smaller, in case of a moveBalance for example
 	totalFee := core.SafeMul(userTx.GetGasPrice(), userTx.GetGasLimit())
 	senderShardID := txProc.shardCoordinator.ComputeId(userTx.SndAddr)
 	if senderShardID != txProc.shardCoordinator.SelfId() {
+		// TODO: check what this does, if the sender is not here, process everything without the moveBalance here?
```

Comment:
yes. as move balance was done in the sender shard.

Reply:
here the totalFee could be smaller than the computeMoveBalanceFee I think

Reply:
totalFee cannot be lower than moveBalanceFee. it is always moveBalance + extra fee.

Reply:
ok